mapcomposer/app/static/translations/it.js

```diff
@@ -942,6 +942,12 @@ GeoExt.Lang.add("it", {
         yAxisLabel: "Asse Y, Aggregazione",
         groupByLabel: "Raggruppa per",
         valueLabel: "Valore, Aggregazione",
-        gaugeMaxText: "Max"
+        gaugeMaxText: "Max",
+        errorFormText: "Devi completare il riempimento della form!",
```

Comment:
`Devi completare la form` or `Devi riempire la form`.
In any case is a little aggressive :) . Enable the button when the form is compiled.